web/styles/main.css

```
@@ -274,6 +285,68 @@ button {
+
+    > * {
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transition: opacity 220ms ease-in-out, transform 500ms ease-in-out;
+    }
+}
+
+.preview-copy-icon-clipboard {
+    width: 14px;
+    fill: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)